indico/modules/rb_new/controllers/backend.py

```diff
@@ -153,19 +164,27 @@ def _process(self, args):
                                                           args['repeat_frequency'], args['repeat_interval'],
                                                           flexibility=0)
         date_range = [dt.isoformat() for dt in date_range]
-        for room_id in availability:
-            data = availability[room_id]
-            data['room'] = rooms_schema.dump(data['room'], many=False).data
-            data['blockings'] = serialize_blockings(data['blockings'])
-            data['nonbookable_periods'] = serialize_nonbookable_periods(data['nonbookable_periods'])
-            data['unbookable_hours'] = serialize_unbookable_hours(data['unbookable_hours'])
-            data.update({k: serialize_occurrences(data[k])
-                         for k in ['candidates', 'pre_bookings', 'bookings', 'conflicts', 'pre_conflicts']})
+
+        for data in availability.viewvalues():
+            # add additional helpful attributes
             data.update({
                 'num_days_available': len(date_range) - len(data['conflicts']),
                 'all_days_available': not data['conflicts']
             })
-        return jsonify_data(flash=False, availability=availability, date_range=date_range)
+
+        return jsonify_data(flash=False,
+                            availability=_serialize_availability(availability),
+                            date_range=date_range)
+
+
+class RHCalendar(RHRoomBookingBase):
+    @use_args({
```

Comment:
`use_kwargs`? you get the two values separately from the dict anyway just below

Reply:
Didn't know that exists, thanks!